### ISSUE TITLE
Apologizes from me, I've started using Environment Variables

### ISSUE THREAD

**Issue by emar** [IMarkoMC/Tokens#8]
Sorry about the acquisition, 
I thought I was been hijacked, when the link from Discord emails took me to GitHub.
Discord kept sending me popups and emails, 11 in total, about my tokens been published on the web,

I knew the original code had the token stored publicly in the config.json, 
I just never thought about using Environment Variable's to make certain info private.
I got the original code from GitHub and managed to port it over to ripl.it and it worked.

In fairness, this lead me to Secrets (Environment Variables)
And I think I've taken care of that now, maybe.

I removed the line that asks for the token, in config.json
"token":"x0x0x0x0x0x0x0x0x0x0x0x0x0x0x0x0x0x0x0x0x0x0",

I clicked Secrets (Environment Variables) in the left menu.
Added "token" to Key field, added bot token to Value field.
Went into index.js and changed client.login("token"); to client.login(process.env.token);

Hit run and it just worked, so as far as I know, the bot token is safe in a hidden .env file in my repl.

Message I got from Discord:

Safety Jim here! It appears that the token for your bot, Pxxxxx Sxxxxx has been posted on the internet. Luckily, our token-scanning gremlins noticed, and have reset your bot's token - hopefully before anyone could have used it maliciously!

I also send a message to the original creator.

**Comment by skidmaster69420** [comment]
Thank you for writing us. We are glad you figured out how to computer. 
Because I still haven't figured out how to do it. 
I'm an unpaid intern.

**Comment by emar** [comment]
@skidmaster69420 
For me the fix was fairly easy, but your code could be different.
The Discord bot token was stored in the config.json file, my app is on repl.it
I removed the whole line that asked for bot token: "token":"{BOT TOKEN}",
Then I clicked the lock icon in the left menu (Secrets Environment Variables)
For the Key field, I entered token. For the Value field, I added the bot token.

**Comment by emar** [comment]
@skidmaster69420 

For me the fix was fairly easy, but your code could be different.
The Discord bot token was stored in the config.json file, my app is on repl.it
I removed the whole line that asked for bot token: "token":"{BOT TOKEN}",
Then I clicked the lock icon in the left menu (Secrets Environment Variables)
For the Key field, I entered "token" (without the two "")
For the Value field, I added the Discord bot token.

**Comment by emar** [comment]
@skidmaster69420 

For me the fix was fairly easy, but your code could be different.
The Discord bot token was stored in the config.json file, my app is on repl.it
I removed the whole line that asked for bot token: "token":"{BOT TOKEN}",
Then I clicked the lock icon in the left menu (Secrets Environment Variables)
For the Key field, I entered "token" (without the two "")
For the Value field, I added the Discord bot token.
Make sure you cluck Save for the newly added Variable.

**Comment by emar** [comment]
@skidmaster69420 

For me the fix was fairly simple, but your code could be different to mine.
The Discord bot token was stored in config.json, my app is hosted on repl.it
I removed the whole line that asked for bot token: "token":"{BOT TOKEN}",
Then I clicked the lock icon in the left menu (Secrets Environment Variables)
For the Key field, I entered "token" (without the two "")
For the Value field, I added the Discord bot token.
Make sure you click Save for the newly added Variable.
